.github/workflows/ci.yml

```diff
@@ -299,10 +299,7 @@ jobs:
               uses: actions/upload-artifact@v4
               with:
                   name: 'river-node-${{ github.job }}'
-                  path: |
-                      ./core/node/run_files/
-                      ./core/xchain/run_files/
-                      !./core/**/bin/**
+                  path: ./core/run_files/
```

Comment:
please keep !bin ignores, otherwise it significantly increase size of collected files